tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -3618,8 +3596,6 @@ public function groupExists(string $group):bool {
 		}
 		if (OcisHelper::isTestingWithGraphApi()) {
 			$base = '/graph/v1.0';
```

Comment:
```suggestion
		}
		$base = '/graph/v1.0';
```

Reply:
```suggestion
		$fullUrl = $this->getBaseUrl() . "/graph/v1.0/groups/$group";
```
@saw-jan can we do this

Reply:
yes, after line `$group = \rawurlencode($group);`